csufmse/student_information_system
# JS and CSS refactoring, standardizing

* pretty much no functional changes, but the classes for tables are now standard as are the styles for the cols and tds.
* refactored the JS in the HTML to common code
* standardized HTML includes of CSS (through base.html now)
* minor (incomplete) improvements to semester -- not testable
* added confirmation on change-password while I was there

Testing:
* verify table cols are reasonable (sorting, names)
* Verify that all rows do something when clicked (jump to the correct entity)
* verify that filter field names (esp range ones) are named

Pretty much just click through the app. No model or functionality changes to explore.

All work is in /schooladmin for now. Still need to address /student, etc -- but that will change views and actual python code so I want to do it separately (#124, #125, #139, #140 task that)

Closes #70, #102, #123.

Now also closes #160 and #147.
